server/handler.go

```diff
@@ -190,3 +190,19 @@ func (h Handler) KillTask(conf *config.Config) http.HandlerFunc {
 		writeJSON(respStatus, body, w)
 	}
 }
+
+func (h Handler) DeleteTask(conf *config.Config) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		id := vars["taskId"]
+		logrus.WithField("task_id", id).Debug("Deleting task")
+		err := h.database.DeleteTask(id)
```

Comment:
That could work.

Disallowing deletion of running tasks should at least not cause unexpected behaviour